tsconfig.base.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "incremental": true /* Enable incremental compilation */,
+    "target": "ES2020" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
+    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
```

Comment:
@ainsleybc messing with the root level config seems to have broken the CI. https://github.com/smartcontractkit/chainlink-solana/runs/5782540100?check_suite_focus=true

Let's scope this to `ts/` because the client and gauntlet code have different requirements.

Reply:
[fc2385f](https://github.com/smartcontractkit/chainlink-solana/pull/224/commits/fc2385f153f4c3616f9ce60e38f88529ee8a67a1)